build/makelib/golang.mk

```diff
@@ -156,6 +156,14 @@ go.test: go.vet go.fmt $(CGO_PREREQS)
 #	@$(GOHOST) test -v -i $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 	@$(GOHOST) test -cover $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 
+.PHONY: go.lint
+go.lint:
```

Comment:
a lot of errors.. I need help from the team to get it down. https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/PR-441/1/pipeline

You can run it locally by using 'make lint'